packages/navie/src/commands/welcome-command.ts

```diff
@@ -0,0 +1,70 @@
+import Command, { CommandRequest, hasCodeSelectionArray } from '../command';
+import CompletionService from '../services/completion-service';
+import { UserContext } from '../user-context';
+import z from 'zod';
+
+export const WelcomeSummary = z.object({
+  activity: z
+    .string()
+    .describe(
+      `The value of this field should fill in the blank to complete the sentence: "It looks like you're _____."`
+    ),
+  suggestions: z
+    .string()
+    .array()
+    .describe(`A list of prompt suggestions for what the user can do next.`),
+});
+
+const WELCOME_SYSTEM_PROMPT = `Your job is to analyze the git diffs provided by the user to determine the task they are working on.
+You will provide a list of prompt suggestions for what the user can ask Navie next. The prompt suggestions should be concise and actionable, referencing specific concepts from the diff. Do not provide any code snippets or other details.
+Prompts should be limited to @explain or @diagram.
```

Comment:
Is it the case that we don’t need to describe Navies capabilities, because the LLM already knows?

Reply:
That's right. I was kind of surprised by this, but I've tested with Sonnet as well and it works without error.